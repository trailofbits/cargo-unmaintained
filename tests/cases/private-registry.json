[
  {
    "name": "crates-io",
    "outdated_deps": [
      {
        "name": "syn",
        "req": "^1.0",
        "version_latest": null,
        "version_used": "1.0.109"
      }
    ],
    "repo_status": "Unnamed",
    "version": "0.1.0"
  }
]
