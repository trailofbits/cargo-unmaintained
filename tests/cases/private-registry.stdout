crates-io (no repository)
    syn (requirement: ^1.0, version used: 1.0.109, latest: [..])
